### ISSUE TITLE
Problem with Angular $templateCache

### ISSUE THREAD

**Issue by ViieeS** [nolimits4web/Swiper#1942]
This is a (multiple allowed):
* [x] bug

* Swiper Version: 2.7.6
* Platform/Target and Browser Versions: macOS, CHROME.

Slider doesn't load images (promonavi.com.br) if I pre-generate Angular templateCache via AMD Loader. Images load only after I resize window.

Now work because I set timeout 500ms before Swiper init. Tried to initSwiper after all images loaded - no luck...

```javascript
$timeout(function () {
	initSwiper();
}, 500);

function initSwiper(){
	var initIterator = 0;
	$('.swiper-container').each(function(){								  
		var $t = $(this);								  

		var index = 'swiper-unique-id-'+initIterator;

		$t.addClass('swiper-'+index + ' initialized').attr('id', index);
		$t.find('.pagination').addClass('pagination-'+index);

		var autoPlayVar = parseInt($t.attr('data-autoplay'));
		var centerVar = parseInt($t.attr('data-center'));
		var simVar = ($t.closest('.circle-description-slide-box').length)?false:true;
		var slidesPerViewVar = $t.attr('data-slides-per-view');
		var xsValue, smValue, mdValue, lgValue;
		if(slidesPerViewVar == 'responsive'){
			slidesPerViewVar = 1;
			xsValue = $t.attr('data-xs-slides');
			smValue = $t.attr('data-sm-slides');
			mdValue = $t.attr('data-md-slides');
			lgValue = $t.attr('data-lg-slides');
		}
		else if(slidesPerViewVar != 'auto'){
			slidesPerViewVar = parseInt(slidesPerViewVar);
		}
		var loopVar = parseInt($t.attr('data-loop'));
		var speedVar = parseInt($t.attr('data-speed'));


		swipers['swiper-'+index] = new Swiper('.swiper-'+index,{
			speed: speedVar,
			pagination: '.pagination-'+index,
			loop: loopVar,
			paginationClickable: true,
			autoplay: autoPlayVar,
			slidesPerView: slidesPerViewVar,
			keyboardControl: true,
			calculateHeight: true, 
			simulateTouch: simVar,
			centeredSlides: centerVar,
			onInit: function(swiper){
				if($t.attr('data-slides-per-view')=='responsive') updateSlidesPerView(xsValue, smValue, mdValue, lgValue, swiper);	
				if($t.find('.swiper-slide').length>swiper.params.slidesPerView) $t.removeClass('hide-pagination');
				else $t.addClass('hide-pagination');
			},
			onSlideChangeStart: function(swiper){
				var activeIndex = (loopVar===true)?swiper.activeIndex:swiper.activeLoopIndex;
				if($t.closest('.testimonials-container').length){
					$t.closest('.testimonials-wrapper').find('.testimonials-icons .entry div.active').removeClass('active');
					$t.closest('.testimonials-wrapper').find('.testimonials-icons .entry div').eq(activeIndex).addClass('active');
				}
				if($t.closest('.block.type-10').length){
					$t.closest('.block.type-10').find('.tab-entry.active').removeClass('active');
					$t.closest('.block.type-10').find('.tab-entry').eq(activeIndex).addClass('active');
				}
			},
			onSlideClick: function(swiper){
				if($t.closest('.circle-slide-box').length) swiper.swipeTo(swiper.clickedSlideIndex);
			},
				#
				onSlideTouch: function (swiper) {
					$(swiper.container).parents('.block.type-13').one('mouseup', function () {
						var activeIndex = Math.round(Math.abs(swiper.positions.current) / swiper.width * swiper.imagesLoaded);
						swiper.swipeTo(activeIndex);
					});
				}
			});
		swipers['swiper-'+index].reInit();
		swipers['swiper-'+index].resizeFix();
		if($t.find('.default-active').length) swipers['swiper-'+index].swipeTo($t.find('.swiper-slide').index($t.find('.default-active')), 0);

		initIterator++;
	});
```

**Comment by nolimits4web** [comment]
Issue is closed because of outdated/irrelevant/not actual/needed more information/inactivity.

If this issue is still actual and reproducible for latest version of Swiper, please create new issue and fill the issue template correctly:
* Clearly describe the issue including steps to reproduce when it is a bug.
* Make sure you fill in the earliest version that you know has the issue.
* Provide live link or JSFiddle/Codepen or website with issue
